editors/vscode/package.json

```diff
@@ -22,7 +22,7 @@
     "oxc"
   ],
   "engines": {
-    "vscode": "^1.95.0"
+    "vscode": "^1.96.0"
```

Comment:
pnpm run build fails because the `@types/vscode` package is not in sync. 

Should we only support the latest engine or tell renovate to ignore the package?

Reply:
probably easiest to only support latest